src/DSharp.Mscorlib/Collections/IList.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Runtime.CompilerServices;
+
+namespace System.Collections
+{
+    //
+    [ScriptImport]
+    [ScriptName("IList")]
```

Comment:
Do we need ScriptName here?

Reply:
Not sure, `ICollection` had it so I assumed it does something under the hood

Reply:
We should be able to remove it. As the transformed name here will the same as the symbolname. It's only really useful to use `ScriptName` where we have different member names